app/models/feature.rb

```diff
@@ -1,13 +1,16 @@
 class Feature < ApplicationRecord
   extend FriendlyId
   friendly_id :name
-  has_and_belongs_to_many :smart_proxies
+
+  has_many :smart_proxy_features, :dependent => :destroy
+  has_many :smart_proxies, :through => :smart_proxy_features
+
```

Comment:
So previously the table name was named `features_smart_proxies` (automatically by Rails) and we are renaming it to `smart_proxy_features` here. Just note to myself, need to check later on.

Reply:
Check.